src/Platform/std3D.c

```diff
@@ -452,9 +442,13 @@ void std3D_SetRenderState(Std3DRenderState rdflags)
     if (std3D_renderState ^ (rdflags & STD3D_RS_FOG_ENABLED))
     {
         if (rdflags & STD3D_RS_FOG_ENABLED)
-            glEnable(GL_FOG);
+        {
+            renderer_setFog(true);
```

Comment:
in your current design, swe1r_functions depends on the functionality in renderer_utils, maybe they should be statically linked to swe1r_functions directly

Reply:
Yes it would be better. I'll use a pragma on GLFW_BACKEND to define the functions and global variables so that its still accessible in the dll side